lib/pvm.maxpat

```diff
@@ -566,36 +566,36 @@
 		}
 ,
 		"dependency_cache" : [ 			{
-				"name" : "pvm_send.maxpat",
-				"bootpath" : "~/Downloads/PVM",
-				"patcherrelativepath" : ".",
-				"type" : "JSON",
+				"name" : "max-init.txt",
+				"bootpath" : "~/Documents/GitHub/PVM",
```

Comment:
What does this path mean, other than LGTM

Reply:
max projects creates those paths for keeping track of the related files. I think these are not set in stone, and that max is clever enough to realize when it is on a different machine. Still, pretty weird that it is not a relative path. I can try to open this branch on another machine to double check at some point. Or maybe you could try it already on the studio iMac? thanks

Reply:
OK, I see, then everything looks good.

Reply:
yes, just confirmed this. if you look at the line right after that it has the relative path. So, we're good!